docs/graphql/manual/queries/simple-object-queries.rst

```diff
@@ -72,3 +72,28 @@ Fetch an object using its primary key
         }
       }
     }
+
+Fetch value from JSON/JSONB column at particular path
+-----------------------------------------------------
+**Example:** Fetch pincode from address for a user where address is a JSON/JSONB column
+
+.. graphiql::
+  :view_only:
+  :query:
+    query {
+      user_by_pk(id: 1) {
+        id
+        name
+        address(path: "$.pincode")
+      }
+    }
```

Comment:
This feels odd without the alias. Maybe we rephrase the intro to make this clear. 

I'll change it off. Will come up with a clear line

Reply:
I have updated the example. Hopefully it should be clear.

Reply:
thanks Rikesh !